starboard/linux/shared/test_filters.py

```diff
@@ -37,7 +37,9 @@
     'player_filter_tests': [test_filter.FILTER_ALL],
 }
 
-_FILTERED_TESTS = {}
```

Comment:
just confirming , this only filters out nplb tests for modular builds .. correct ?

Reply:
Correct